config.json

```diff
@@ -110,6 +110,12 @@
       ]
     },
     {
+      "slug": "bracket-push",
+      "difficulty": 3,
```

Comment:
In that case, I'll include `Stack` in the list of topics and increase its difficulty to **5**. I agree it shouldn't go any higher than that.